scripts/apps/authoring/authoring/directives/AuthoringDirective.ts

```diff
@@ -226,6 +226,11 @@ export function AuthoringDirective(superdesk, superdeskFlags, authoringWorkspace
                         $scope.item.associations = _.cloneDeep(res.associations);
                     }
 
+                    // cloneDeep fields_meta so that diff can be calculated on save
+                    if (res.fields_meta) {
```

Comment:
Why not update the entire object instead of doing it field by field? When such field-specific conditions are added, it becomes impossible to know what needs to be updated for a new field to work properly and we same issue will come back again and again.

It looks it already happened with associations code you see above. Looks like `angular.extend($scope.item, res);` attempted to address this, but I think `angular.extend` is shallow. Try this instead: `$scope.item = cloneDeep(res);`.

Also, remove the conditional updating associations explicitly and re-test `SDESK-4104`
Remove the conditional referring to `res.cropData` and `item.hasCrops`. I don't see any usages of these.